telosnetwork/teloscan
# Update explorer links

## Notes
Addresses issue #71 

This PR updates netlify environment vars to point network explorer urls to telos.net

Also removed Transaction.vue, which references the updated environment var but marked for deletion by @poplexity 

## Testing
**note** that if you have an untracked `.env` file in your local repo (based on .`.env.example` as suggested in the README), it will need to be updated. If you have no custom modifications, you can run the following from the project root:
    `cp ./.env.example ./.env`

- start up dev server: `npm run dev`
- go to native account http://localhost:8080/address/0xbe4d0bf0018f1e929cb4f188468c297d6669b051
- click on account link "jesse.tcd"
    - you should be sent to https://explorer.telos.net/account/jesse.tcd
    - compare to https://www.teloscan.io/address/0xbe4d0bf0018f1e929cb4f188468c297d6669b051 which links to EOS authority
